Jupyter Notebook/Computer Vision API Example.ipynb

```diff
@@ -50,10 +50,9 @@
    },
    "outputs": [],
    "source": [
-    "# Variables\n",
-    "_region = 'westcentralus' #Here you enter the region of your subscription\n",
-    "_url = 'https://{}.api.cognitive.microsoft.com/vision/v2.0/analyze'.format(_region)\n",
-    "_key = None #Here you have to paste your primary key\n",
+    "# Variables\n"
+    "_url = <YOUR_ENDPOINT> # Here you have to paste your endpoint\n,
```

Comment:
I don't like the fact that most of the URL is lost with this change.  I'd prefer that it looked something like:

```
_endpoint = 'YOUR_ENDPOINT' # Copy this from the Azure Portal; e.g. https://yourname.cognitiveservices.azure.com/
_url = '{}vision/v2.0/analyze'.format(_endpoint)\n",
```

Reply:
Sure, that works. Yeah, I wasn't sure if you needed to append the service/version etc. Good call. That change works. Thanks. Also, I added edits to the OCR file, I left in the appendage, but change it more like this one if you want.

Reply:
I'm a bit confused.  The portal only contains the base URL, so the old OCR notebook had it correctly.  I'd like to see the Compter Vision notebook to look like how OCR used to look like.
<img width="274" alt="Screen Shot 2019-09-26 at 4 04 31 PM" src="https://user-images.githubusercontent.com/14881441/65730697-8cd4d800-e077-11e9-8723-3249f4737c1f.png">

Reply:
True, I checked my Computer Vision resource and it's just the base. (Not all services provide base, but I guess some do). @erhopf Were you aware it's just the base URL for CV in Azure?